bot/cogs/source.py

```diff
@@ -94,22 +93,22 @@ async def build_embed(self, link: str, source_object: SourceType, loc: str, firs
         """Build embed based on source object."""
         if isinstance(source_object, HelpCommand):
             title = "Help"
-            description = source_object.__doc__
+            description = source_object.__doc__.splitlines()[1]
         elif isinstance(source_object, Command):
             if source_object.cog_name == "Alias":
                 cmd_name = source_object.callback.__name__.replace("_alias", "")
                 cmd = self.bot.get_command(cmd_name.replace("_", " "))
-                description = cmd.help
+                description = cmd.short_doc
             else:
-                description = source_object.help
+                description = source_object.short_doc
 
             title = source_object.qualified_name
         elif isinstance(source_object, str):
             title = f"Tag: {source_object.split('/')[-1].split('.')[0]}"
             description = ""
         else:
             title = source_object.qualified_name
-            description = source_object.description
+            description = source_object.description.splitlines()[0]
```

Comment:
Does this account for the different styles?

```py
"""Hello"""
```

vs.

```py
"""
Hello
"""

Reply:
Yes, this account both styles.